core/shared/src/main/scala/monocle/function/Index.scala

```diff
@@ -45,6 +46,13 @@ object Index extends IndexFunctions {
   /** Std instances */
   /** *********************************************************************************************
     */
+  implicit def seqIndex[A, S[B] <: SeqOps[B, S, S[B]]]: Index[S[A], Int, A] =
```

Comment:
Do we need the implicit for List, Vector, etc? 
Can you also add a test for Seq.

Reply:
> Do we need the implicit for List, Vector, etc?
OK, we don't need specific implementations for them.
But should we remove them? It would break backward compatibility.

Reply:
> The question is does Scala select the most precise implicit?

Yes, it does.